src/core/server/http/http_service.ts

```diff
@@ -117,6 +116,24 @@ export class HttpService
       ) => this.requestHandlerContext!.registerContext(pluginOpaqueId, contextName, provider),
     };
 
+    if (notReadySetup) {
+      this.log.warn('haz not ready');
+      // We cannot use the real context container since the core services may not yet be ready
+      const fakeContext: RequestHandlerContextContainer = deps.context.createContextContainer();
+      this.internalSetup.notReadyServer = {
```

Comment:
Minor: let's avoid mutating `internalSetup` object. We can create the property during the object instantiation.

Reply:
Resolved in [`efbbc7c` (#79012)](https://github.com/elastic/kibana/pull/79012/commits/efbbc7cad755d02fd9dbfee16b2f6f8aece1e2a3)